### ISSUE TITLE
Add AppVeyor Windows build to PR checks

### ISSUE THREAD

**Issue by oleksiyskononenko** [h2oai/datatable#2308]
Since now datatable compiles on Windows, each new PR should be checked on AppVeyor for not breaking the MSVC compatibility. This can be done with building datatable on AppVeyor and a subsequent test to import datatable module in Python. Once #2301 is done, we should also run a full cycle of tests on AppVeyor and produce a wheel, providing full support of datatable on Windows (#1114).
